CHANGELOG.next.asciidoc

```diff
@@ -63,6 +63,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Add support of Graphite series 1.1.0+ tagging extension for statsd module. {pull}39619[39619]
 - Remove fallback to the node limit for the `kubernetes.pod.cpu.usage.limit.pct` and `kubernetes.pod.memory.usage.limit.pct` metrics calculation
 - Add support for Kibana status metricset in v8 format {pull}40275[40275]
+- Add new metrics for the Vsphere Datastore metricset. {pull}40441[40441]
```

Comment:
```suggestion
- Update metrics for the Vsphere Datastore metricset. {pull}40441[40441]
```

Reply:
As discussed we are keeping it as it is since we are adding new metrics.